packages/tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,301 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Popover, { PopoverProps, Align, Justify } from '@leafygreen-ui/popover';
+import { useEventListener } from '@leafygreen-ui/hooks';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { transparentize } from 'polished';
+import debounce from 'lodash/debounce';
+import { trianglePosition } from './tooltipUtils';
+
+export const TriggerEvent = {
```

Comment:
Might this be a bit overkill? JS relies on the plain strings (e.g. `'click'`) when creating event listeners.

Reply:
Perhaps 🤷‍♀

Reply:
It is clean though, and I use in multiple places.